### ISSUE TITLE
Amstelvar Alpha with blending shown

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/variable-type-tools#76]
I just noticed that Amstelvar Alpha is no longer preloaded in https://typetools.typenetwork.com/ and so the blending that was being shown, where dragging the registered axes showed the parametric axes changing, is no longer possible. 

Please add this back :)

**Comment by davelab6** [comment]
Perhaps there should be a checkbox/toggle in the sidebar Show section for "Alpha fonts" to include Amstelvar Alpha, Roboto Delta, Roboto Delta v2, etc?

**Comment by chrissam42** [comment]
I wonder if a better solution than adding back Amstel Alpha would be getting the composite axis math. @sannorozco are there good blended values for Amstelvar Roman (and/or Italic)?

**Comment by dberlow** [comment]
All the axis values in a sheet.
https://docs.google.com/spreadsheets/d/1L1Cy2Y1JFOl32nuevTkzcxjNsSTlLGn_oiRLkMnyonY/edit#gid=291256020

also instructions for how to use, from measurements to running.

On Jun 22, 2020, at 6:22 PM, Chris Lewis <notifications@github.com> wrote:

﻿

I wonder if a better solution than adding back Amstel Alpha would be
getting the composite axis math. @sannorozco <https://github.com/sannorozco>
are there good blended values for Amstelvar Roman (and/or Italic)?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/TypeNetwork/variable-type-tools/issues/76#issuecomment-647798660>,
or unsubscribe

**Comment by chrissam42** [comment]
I've copied in values from the spreadsheet for Amstelvar Roman.

**Comment by dberlow** [comment]
Chris: Can you check and see why not all the blue dots are moving when the values are changing.

SO, DJR: Also, I know along opsz, that YTLC changes, but the values are not.  It’s in “index ref” and in the “axes” sheets. But there is an error in the script that’s leaving that value, out of the “measurement” sheet? I do not see any columns for YTLC.

On Jul 8, 2020, at 10:21 PM, Chris Lewis <notifications@github.com> wrote:

﻿
I've copied in values from the spreadsheet for Amstelvar Roman.

**Comment by dberlow** [comment]
Dave,

We’re working out some bugs along the line, ending with this working on any font with index refs, I.e. paramaroundup measurements. Then, the latest versions of everything we’re working on will always work to show the correct parametric values.

GRAD will not show correct values as it is though,since it was revalued to range -1, 0, 1. I’d be interested in keeping this for several reasons. But I’d also like to experiment with adding a second GRAD axis, with no gvars, but with parametric values. I think underware showed that the first axis will be shown to users, -1,0,1, the effects of the two axes’ gvars will be merged, and our type apps can be trained to show the second value on a slider.

Or both values could be on one slider?

Like a ruler with metric on one side and inches on the other. This could be better for dealing with the grade, wght, and wdth axes, saving UI space and easing the burden of watching one slider while moving another, as e.g. wght and XOPQ could be far apart in the UI.



On May 5, 2020, at 12:58 PM, Dave Crossland <notifications@github.com> wrote:

﻿
I just noticed that Amstelvar Alpha is no longer preloaded in https://typetools.typenetwork.com/ and so the blending that was being shown, where dragging the registered axes showed the parametric axes changing, is no longer possible.

Please add this back :)

**Comment by davelab6** [comment]
Axis duplicates seem buggy. I look forward to your test cases and
experiments.

**Comment by chrissam42** [comment]
I've added a general solution for pulling in the parametric-to-registered axis spreadsheets when they exist. See new `.composites.txt` files for Amstelvar Roman, Italic, and Roboto Flex. 

This is still relatively manual. The Google Sheets need to be "published to web" (Measurements sheet shared as .tsv) and then a file created in typetools to reverence the published URL.

Also, there is a Google Docs bug that kills some of these published URLs by not including CORS (Access-Control-Allow-Origin) headers in redirected links. I first noticed this [a few weeks ago](https://github.com/TypeNetwork/Opentype-1.8-Axis-Proposal/issues/68) and it seems it will keep rearing its head.
